### ISSUE TITLE
Can I add my retriever api? 

### ISSUE THREAD

**Issue by snarb** [mendableai/rag-arena#10]
Can we add custom retrievers? (closed source with api)

**Comment by leonardogrig** [comment]
Sure! But do you mean to the repository or in the actual application (ragarena.com)?

**Comment by snarb** [comment]
@nickscamara yes, we have developed  a graph based retriever that is very different from standart implementations and beat all standart retrievers with significant gap on all datasets that we have tested. Want to test it on your ragarena.com leaderboard. But how?

**Comment by leonardogrig** [comment]
Take a look at the python_service folder. We use it as a service for retrievers, it could be implemented similar to the retrievers we are already using. Add me on Slack using **leonardogrig@gmail.com** to discuss further.
